auto_round/quantizer.py

```diff
@@ -83,8 +84,8 @@ def quant_weight_asym(weight, num_bits=4, v=0, min_scale=1.0, max_scale=1.0, sca
     return scale * (q - zp), scale, zp
 
 
-def quant_weight_sym(weight, num_bits=4, v=0, min_scale=1.0, max_scale=1.0, scale_dtype=torch.float16, weight_min=None,
-                     weight_max=None):
+def quant_tensor_sym(weight, num_bits=4, v=0, min_scale=1.0, max_scale=1.0, scale_dtype=torch.float16, weight_min=None,
+                     weight_max=None, scale_min=0.0):
```

Comment:
cloud call it limit_scale or something else makes it less likely to be confused with min_sacle？